### PR TITLE
Fix lookups that were using get_db()

### PR DESCRIPTION
**Pull request by dannyroberts**
Need this before we can flip to users db

### CODE REVIEW

File: corehq/apps/hqwebapp/models.py [hunk 1]
```diff
@@ -664,8 +664,9 @@ def make_app_title(cls, app_name, doc_type):
     def dropdown_items(self):
         # todo async refresh submenu when on the applications page and
         # you change the application name
+        from corehq.apps.app_manager.models import Application
```
Comment (dimagimon) [inline corehq/apps/hqwebapp/models.py]: F401 'Application' imported but unused

File: corehq/apps/hqwebapp/models.py [hunk 2]
```diff
@@ -664,8 +664,9 @@ def make_app_title(cls, app_name, doc_type):
...
         key = [self.domain]
-        apps = get_db().view('app_manager/applications_brief',
+        apps = Applcation.get_db().view('app_manager/applications_brief',
```
Comment (dimagimon) [inline corehq/apps/hqwebapp/models.py]: F821 undefined name 'Applcation'

File: corehq/apps/sms/util.py [hunk 3]
```diff
@@ -5,7 +5,7 @@
 import datetime
 
 from dimagi.utils.couch.database import get_db
-from corehq.apps.users.models import CouchUser
+from corehq.apps.users.models import CouchUser, CommCareUser
```
Comment (dimagimon) [inline corehq/apps/sms/util.py]: F401 'CommCareUser' imported but unused

File: corehq/apps/hqwebapp/views.py [hunk 4]
```diff
@@ -988,21 +988,15 @@ def deal_with_couch_doc(doc):
...
-    except ResourceNotFound:
-        pass
-    else:
-        return deal_with_couch_doc(doc)
-
-    try:
-        doc = Repeater.get_db().get(query)
-    except ResourceNotFound:
-        pass
+    for db_name in (None, 'users', 'receiverwrapper', 'meta'):
```
Comment (esoergel) [inline corehq/apps/hqwebapp/views.py]: nice

Comment (esoergel) [inline corehq/apps/hqwebapp/views.py]: Noah did something similar for `doc_in_es`

**Pull request merged**